src/Driver/IBMDB2/Statement.php

```
@@ -211,8 +215,23 @@ private function createTemporaryFile()
      */
     private function copyStreamToStream($source, $target): void
     {
+        $resetTo = false;
+        try {
+            if (stream_get_meta_data($source)['seekable']) {
+                $resetTo = ftell($source);
+            }
+        } catch (Throwable $e) {
+            // Swallow
+        }
```

What kind of error are we expecting here? We should avoid catching and ignoring arbitrary errors.

---

I'm just being defensive. I can't guarantee the kind of stream passed in. AFAIK stream_get_meta_data can fail in strange ways depending on the stream type. Yes, this is an arbitrary error, but it is also a finally for a very limited number of requests for information, not for a big piece of code or any calls that change data.

---

Okay, but if the stream errors, I as a developer wanna know. An error that is thrown away is close to impossible to debug. And if the stream errors, I'd rather fail hard than writing garbage to the database.

Please remove those try/catch blocks.

---

Try catch blocks are gone for all uses of this construct in the latest commit.